services/media-pipeline-service/src/routes/transcript.routes.ts

```
@@ -0,0 +1,366 @@
+      }
+
+      const useRedacted = redacted === 'true';
+
+      switch (format) {
+        case 'txt':
+          const textContent = transcript.utterances
+            .map((u) => {
+              const content = useRedacted && u.contentRedacted ? u.contentRedacted : u.content;
+              return `[${u.speakerLabel || 'Unknown'}]: ${content}`;
+            })
+            .join('\n\n');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)